### PR TITLE
Remove nullable macros

### PR DESCRIPTION
**Pull request by realf**

### CODE REVIEW

File: ViperFeature/VIPER.xctemplate/Interactors/___FILEBASENAME___InteractorProtocol.h [hunk 1]
```diff
@@ -8,10 +8,6 @@
 
 #import <Foundation/Foundation.h>
 
-NS_ASSUME_NONNULL_BEGIN
-
 @protocol ___FILEBASENAMEASIDENTIFIER___InteractorProtocol <NSObject>
 
 @end
```
Comment (nburyak) [inline ViperFeature/VIPER.xctemplate/Interactors/___FILEBASENAME___InteractorProtocol.h]: It's likely to add newlines.

File: ViperFeature/VIPER.xctemplate/Views/___FILEBASENAME___ViewProtocol.h [hunk 2]
```diff
@@ -8,10 +8,6 @@
 
 #import <Foundation/Foundation.h>
 
-NS_ASSUME_NONNULL_BEGIN
-
 @protocol ___FILEBASENAMEASIDENTIFIER___ViewProtocol <NSObject>
 
 @end
```
Comment (nburyak) [inline ViperFeature/VIPER.xctemplate/Views/___FILEBASENAME___ViewProtocol.h]: It's likely to add newlines.

**Comment by nburyak** [comment]
Thank you! Please, merge 👍

File: ViperFeature/VIPER.xctemplate/Interactors/___FILEBASENAME___InteractorProtocol.h (hunk 1 above)
Comment (realf) [inline ViperFeature/VIPER.xctemplate/Interactors/___FILEBASENAME___InteractorProtocol.h]: \n is there

File: ViperFeature/VIPER.xctemplate/Views/___FILEBASENAME___ViewProtocol.h (hunk 2 above)
Comment (realf) [inline ViperFeature/VIPER.xctemplate/Views/___FILEBASENAME___ViewProtocol.h]: same here

**Comment by realf** [comment]
Please merge yourself, I don't have permissions to merge in your repo.

**Comment by realf** [comment]
BTW, couldn't manage to create folder groups using TemplateInfo.plist :(

**Pull request merged**